model/task_lifecycle.go

```diff
@@ -852,6 +852,13 @@ func MarkStart(t *task.Task, updates *StatusChanges) error {
 		}
 	}
 
+	// update task unblocked field
+	err = t.CalculateUnblockedTime()
+
+	if err != nil {
+		return errors.Wrapf(err, "error calculating unblocked_time for task %s", t.Id)
```

Comment:
nit: '%s'

Reply:
I don't understand what the problem is.

Reply:
ah, we generally wrap IDs with ' ' quotes for readability (i.e. for task 'task-1234') (but this doesn't affect anything hence why it's a nit)